src/ui/profile_preferences.ui

```diff
@@ -2153,6 +2153,19 @@ you can use it but there could be issues with aligning columns of text</string>
             </layout>
            </widget>
           </item>
+          <item row="1" column="0">
+           <widget class="QCheckBox" name="mMapperShowBorders">
```

Comment:
Might I suggest a rename to `mapperShowBorders` or even:

```suggestion
           <widget class="QCheckBox" name="checkBox_mapperShowRoomBorders">
```

it is useful to identify the type of widget that one is working with in the associated `dlgProfilePreferences` class and what it is showing borders on...! :wink: